docker/airflow/Dockerfile

```diff
@@ -33,21 +34,19 @@ ENV AIRFLOW__LOGGING__REMOTE_LOG_CONN_ID=aws_default
 ENV AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER=s3://openverse-airflow-logs
 
 
-RUN apt-get update && apt-get -yqq upgrade && apt-get -yqq install \
+USER root
+RUN apt-get update && apt-get -yqq install \
```

Comment:
Why no more `upgrade`?

Reply:
There were some serious version conflicts/difficulties in doing this with the Airflow image. I should have noted exactly what the problem was, I can try and re-add it to note the specific error I received.